### DISCUSSION TITLE
Use of python

### DISCUSSION THREAD

**Discussion by avinas-dwivedi** [Coronasafe/life#154] [Q&A] [answered]
Hi Team,

Can anyone tell me why we used python, instead of calling Aittable API's directly?

Thanks
Avinas

**Comment by vigneshhari** [comment, marked as answer]
The deployed pages are static, the python script scraps the data into a JSON and then the js framework builds static pages based on those JSON files, it's extremely simple and fast and does not require a backend at all.
